examples/geo_schelling/server.py

```diff
@@ -19,10 +19,8 @@ def render(self, model):
 
 
 model_params = {
-    "density": UserSettableParameter("slider", "Agent density", 0.6, 0.1, 1.0, 0.1),
-    "minority_pc": UserSettableParameter(
-        "slider", "Fraction minority", 0.2, 0.00, 1.0, 0.05
-    ),
+    "density": Slider("Agent density", 0.6, 0.1, 1.0, 0.1),
```

Comment:
This should have been `mesa.visualization.Slider`. We flatten every viz stuff into `mesa.visualization`.

Reply:
Thanks! Didn't know that.

Strangely I got `ImportError: cannot import name 'Slider' from 'mesa.visualization'` when doing `from mesa.visualization import Slider`, but `mesa.visualization.Slider` works. Is this expected?

Reply:
Yes, that is expected.

Reply:
Ok, this is interesting. I've changed it to `mesa.visualization.Slider`.